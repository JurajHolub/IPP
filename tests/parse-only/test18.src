.IPPcode19
DPRINT string@to_stderr
BREAK
