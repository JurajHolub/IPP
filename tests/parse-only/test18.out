<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode19">
  <instruction order="1" opcode="DPRINT">
    <arg1 type="string">to_stderr</arg1>
  </instruction>
  <instruction order="2" opcode="BREAK"/>
</program>

